Pipfile

```diff
@@ -0,0 +1,11 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+
```

Comment:
I think we should add sourced-ml at least. 
And all other requirements too if we have more.

Reply:
@zurk I've added `sourced-ml` and `tensorflow` for now.